2025-06-24 23:15:59,711 [INFO] app: Aplicação iniciada!
2025-06-24 23:21:53,756 [INFO] app: Aplicação iniciada!
2025-06-24 23:54:56,656 [INFO] app: Aplicação iniciada!
2025-06-24 23:54:57,959 [INFO] app: Aplicação iniciada!
2025-06-25 20:46:01,439 [INFO] app: Aplicação iniciada!
2025-06-25 20:46:03,293 [INFO] app: Aplicação iniciada!
2025-06-25 20:47:23,195 [INFO] app: Aplicação iniciada!
2025-06-25 20:47:32,554 [INFO] app: E-mail enviado para sabores@ilha.co.ao | Assunto: Sua assinatura expirou | Corpo: Olá Sabores da Ilha,

Sua assinatura foi desativada automaticamente por expiração no dia 20/06/2025.

Para reativar seus serviços, acesse o painel e renove sua assinatura.

Equipe QrMenuX @ NextKode.
2025-06-25 20:47:35,743 [INFO] app: E-mail enviado para vegan@banda.ao | Assunto: Sua assinatura expirou | Corpo: Olá Vegano na Banda,

Sua assinatura foi desativada automaticamente por expiração no dia 24/06/2025.

Para reativar seus serviços, acesse o painel e renove sua assinatura.

Equipe QrMenuX @ NextKode.
2025-06-25 20:52:59,852 [INFO] app: Aplicação iniciada!
2025-06-25 20:58:08,245 [INFO] app: Aplicação iniciada!
2025-06-25 21:02:32,002 [INFO] app: Aplicação iniciada!
2025-06-25 21:10:16,256 [INFO] app: Aplicação iniciada!
2025-06-25 21:23:58,020 [INFO] app: Aplicação iniciada!
2025-06-25 21:24:07,832 [INFO] app: Aplicação iniciada!
2025-06-25 21:24:19,472 [ERROR] app: Erro 404 - /admin/aa - 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-06-25 21:24:19,492 [ERROR] app: Traceback (most recent call last):
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/app.py", line 891, in dispatch_request
    self.raise_routing_exception(req)
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/app.py", line 500, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 629, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.

2025-06-25 21:25:01,605 [ERROR] app: Erro 404 - /admin/aa - 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-06-25 21:25:01,607 [ERROR] app: Traceback (most recent call last):
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/app.py", line 891, in dispatch_request
    self.raise_routing_exception(req)
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/app.py", line 500, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 629, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.

2025-06-25 21:25:46,818 [ERROR] app: Erro 404 - /admin/aa - 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-06-25 21:25:46,821 [ERROR] app: Traceback (most recent call last):
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/app.py", line 891, in dispatch_request
    self.raise_routing_exception(req)
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/app.py", line 500, in raise_routing_exception
    raise request.routing_exception  # type: ignore[misc]
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/flask/ctx.py", line 362, in match_request
    result = self.url_adapter.match(return_rule=True)  # type: ignore
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/eric/PycharmProjects/QrMenuX/.venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 629, in match
    raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.

